code/modules/cm_aliens/XenoStructures.dm

```diff
@@ -844,6 +844,159 @@
 
 	return ..()
 
+/obj/effect/alien/resin/destroyer_cocoon
+	name = "alien cocoon"
+	desc = "A large pulsating cocoon."
+	icon = 'icons/obj/structures/alien/xenoDestroyerHatchery.dmi'
+	icon_state = "static"
+	health = 3600
+
+	// The mob picked as a candidate to be the destroyer
```

Comment:
```suggestion
	/// The mob picked as a candidate to be the destroyer
```
See https://github.com/cmss13-devs/cmss13/blob/master/.github/guides/AUTODOC.md `///` and `/**` form special comments that VSC and the reference https://docs.cm-ss13.com/ display.